libethereum/BlockChainSync.cpp

```diff
@@ -908,9 +902,8 @@ void BlockChainSync::onPeerNewHashes(std::shared_ptr<EthereumPeer> _peer, std::v
     for (auto const& p: _hashes)
     {
         h256 const& h = p.first;
-        _peer->addRating(1);
-        DEV_GUARDED(_peer->x_knownBlocks)
-            _peer->m_knownBlocks.insert(h);
+        m_host.capabilityHost().addRating(_peerID, 1);
```

Comment:
Maybe rename `addRating` to `updateRating`?

Reply:
done